public/locales/en/translation.json

```diff
@@ -13,5 +13,26 @@
   },
   "ar": {
     "1": "Arabic"
+  },
+  "whoAreWe": {
+    "header": "who are we?"
+  },
+  "aboutPart": {
+    "bigText": "We are",
```

Comment:
you don't have to fix this now but you may have to fix it eventually, i just want to point out that assuming "We are" or that the first part of the sentence can be emphasis is universal per language (of course you can already imagine this is different in turkish). google mdn localization best practices and read about it, there are a few ways you can handle such a scenario